main.py

```diff
@@ -121,31 +121,33 @@ def registration():
         with open("access_token.txt") as f:
             if access_token != f.read():
                 print("bad token")
-                flash('Incorrect access token', 'error')
+                flash("Incorrect access token", "error")
                 return render_template("registration.html")
 
         if "@" not in email:
-            flash('Incorrect email', 'error')
+            flash("Incorrect email", "error")
             return render_template("registration.html")
 
         if "." not in email:
-            flash('Incorrect email', 'error')
+            flash("Incorrect email", "error")
             return render_template("registration.html")
 
         if len(username) > 32:
-            flash('Incorrect username', 'error')
+            flash("Incorrect username", "error")
             return render_template("registration.html")
 
         if "-" not in unit_number or len(unit_number) > 5:
-            flash('Incorrect unit-number', 'error')
+            flash("Incorrect unit-number", "error")
             return render_template("registration.html")
 
         success = db.create_applicant(
             username, email, unit_number, password, is_dispatch, is_civilian, is_police
```

Comment:
Lets try to run [`black`](https://github.com/ambv/black`) on this file to clean up some of the formatting. I believe `black` would typically put all of these parameters on their own line.